lib/pflua/src/pf/utils.lua

```diff
@@ -11,6 +11,11 @@ struct pflua_timeval {
 int gettimeofday(struct pflua_timeval *tv, struct timezone *tz);
 ]]
 
+-- Additional function arguments other than 'len' and 'P' that the
+-- compiled function may accept (this needs to be tracked in several
+-- pflua passes, which is why the data is kept here)
+extra_args = {}
```

Comment:
Generalizing this to include `len` is a good idea, thanks. I implemented the change and then localized the variable in each module.